lib/core/Doc.js

```diff
@@ -60,11 +60,12 @@ class Doc extends React.Component {
         </a>
       );
     }
+
     return (
       <div className="post">
         <header className="postHeader">
           {editLink}
-          <h1>{this.props.title}</h1>
+          {this.props.hideTitle && <h1>{this.props.title}</h1>}
```

Comment:
Wait. Shouldn't this be 

```
!this.props.hideTitle
```

We only want to show this if we don't have hideTitle, right?